src/Core/Settings.h

```diff
@@ -897,6 +897,7 @@ class IColumn;
     M(UInt64, extract_key_value_pairs_max_pairs_per_row, 1000, "Max number of pairs that can be produced by the `extractKeyValuePairs` function. Used as a safeguard against consuming too much memory.", 0) ALIAS(extract_kvp_max_pairs_per_row) \
     M(Bool, restore_replace_external_engines_to_null, false, "Replace all the external table engines to Null on restore. Useful for testing purposes", 0) \
     M(Bool, restore_replace_external_table_functions_to_null, false, "Replace all table functions to Null on restore. Useful for testing purposes", 0) \
+    M(Bool, create_if_not_exists, false, "Enable IF NOT EXISTS for CREATE statements by default", 0) \
```

Comment:
I'm not super convinced about this setting name, but I could not come up with something better. Maybe something like `implicit_create_if_not_exists`? WDYT

Reply:
I like `default_create_if_not_exists` better, but I also don't see much of a problem with `create_if_not_exists`. I can only imagine it being a problem if the name was misleading to some, but I can't think of another way developers might interpret the behavior based on the current name. Since the setting is for convenience, I'd like to avoid making the name longer unnecessarily. What do you think? I'd be happy to change the name to something else if we decide something else is better.

Reply:
i don't see a better (and ATST concise) name neither. let's leave it as is.